src/services/offerVisibilityService.js

```diff
@@ -0,0 +1,72 @@
+import config from "../config";
```

Comment:
I think we should just merge this file with the offerSearchService (and call it offerService.js) which would contain offer-related operations.